### DISCUSSION TITLE
Nushell icon with Greek nu?

### DISCUSSION THREAD

**Discussion by connorjs** [nushell/nushell#10310] [Ideas]
TL;DR - What are folks’ thoughts on branding Nushell’s prompt symbol with a rotated lowercase Greek nu? _Some unicode nu’s follow._
- ν 
- 𝜈

---

I was adding a shell segment to my Oh My Posh configuration so that my prompt showed my active shell (pwsh, nu, or fish mainly). I have a nerd font installed, so I easily found powershell + fish (and cmd and bash), but nushell did not have an icon. I ended up choosing a grid icon given the table focus + dataframe feature, but I wanted more.

That’s when I thought about Greek nu for Nushell. I looked at capital and lowercase, and realized that the lowercase, rotated 90°, has strong parallels with the `>` shell prompt symbol (also `❯`). **So, what are folks’ thoughts on that?**

I also came across [#2872](https://github.com/nushell/nushell/issues/2872), which led to the current (I think) branding. Thus, I decided to start a discussion instead of an issue.

**Comment by fdncred** [comment]
We don't really have official branding so adding it would be cool to see it as an icon for the app, or at least see some app icon designs. I think the only way to get it into the prompt is if that rotated symbol is part of the font. Not to mention, I'm not sure how we get people to add it to their nerd font packages. If it were possible, I think it would be cool.

**Reply by connorjs** [reply]
> If it were possible, I think it would be cool.

Thanks for the encouragement! I may look at this for fun to learn more about nerd font icons. Plus other icon design.
